features/Blocklist.php

```diff
@@ -173,35 +169,22 @@ private function _getBlocklists() : array {
 	}
 
 	/**
-	 * Register a hook to handle the upload/removal of blocklists
-	 * @see PKPComponentRouter::route()
+	 * Register a hook for LoadComponentHandler
 	 */
-	/*
-	private function _registerBlocklistFileHandler() : void {
-		Hook::add('LoadComponentHandler', function ($hook, $args) {
-			$component = &$args[0];
-			$operation = $args[1];
-			if (!in_array($operation, ['deleteBlocklist', 'uploadBlocklist'])) {
-				return;
-			}
-
-			define('HANDLER_CLASS', 'BlocklistHandler');
-			$component = 'plugins.generic.betterPassword.handlers.BlocklistHandler';
-			import($component);
-			return true;
-		});
-	}
-	*/
 	public function register() {
-		//$success = parent::register($category, $path, $mainContextId);
 		Hook::add('LoadComponentHandler', [$this, 'setComponentHandler']);
 	}
 
+	/**
+	 * Create a new BlocklistHandler
+	 * @param string $hookname The hook name
+	 * @param array $args Arguments of the hook
+	 * @return bool true if Blocklist handler successfully created
```

Comment:
The true/false return here is more properly described as returning true if the plugin is the final handler for the hook, false if other hooks should be called to intervene.